doc/api/crypto.md

```diff
@@ -935,10 +935,15 @@ Calculates the signature on all the data passed through using either
 
 The `private_key` argument can be an object or a string. If `private_key` is a
 string, it is treated as a raw key with no passphrase. If `private_key` is an
-object, it is interpreted as a hash containing two properties:
+object, it is interpreted as a hash containing some of these properties:
 
-* `key` : {String} - PEM encoded private key
-* `passphrase` : {String} - passphrase for the private key
+* `key`: {string} - PEM encoded private key (required)
+* `passphrase`: {string} - passphrase for the private key
+* `padding`: {String} - RSA padding, either `RSA_PKCS1_PADDING` (default) or
+  `RSA_PKCS1_PSS_PADDING`
+* `saltLength`: {number} - salt length for RSASSA-PSS. The special value
```

Comment:
`padding` and `saltLength` should both be `integer`, right? (Or at least `number`. But `integer` seems to be what’s coming out of the discussion @ https://github.com/nodejs/node/pull/11697)